builds/e2e/templates/e2e-setup-base-image-update-release.yaml

```diff
@@ -0,0 +1,235 @@
+        testResultCoordinatorImage = "$imagePrefix-test-result-coordinator:$versionInfo";
+        metricsValidatorImage = "$imagePrefix-metrics-validator:$versionInfo";
+        numberLoggerImage = "$imagePrefix-number-logger:$versionInfo";
+        edgeAgentBootstrapImage = "$imagePrefix-agent-bootstrap-e2e-$(os)-$(arch)";
+        genericMqttTesterImage = "$imagePrefix-generic-mqtt-tester:$versionInfo"
+        registries = @(
+          @{
+            address = '$(registry.address)';
+            username = '$(registry.username)';
+          },
+          @{
+            address = '$(contenttrust.address)';
+            username = '$(contenttrust.username)';
+          }
+        );
+        caCertScriptPath = "$caCertScriptPath";
+        rootCaCertificatePath = "$rootCaCertificatePath";
+        rootCaPrivateKeyPath = "$rootCaPrivateKeyPath";
+        logFile = Join-Path '$(binDir)' 'testoutput.log';
+        verbose = '$(verbose)';
+        getSupportBundle = 'true';
+        manifestSigningDeploymentPath = "$testManifestSigningDeploymentPath";
+        manifestSigningSignedDeploymentPath = "$testManifestSigningSignedDeploymentPath";
+        manifestSigningGoodRootCaPath = "$testManifestSigningGoodRootCaPath";
+        manifestSigningBadRootCaPath = "$testManifestSigningBadRootCaPath";
+        manifestSigningDefaultLaunchSettings = "$testManifestSigningDefaultLaunchSettings";
+        manifestSigningLaunchSettingsPath = "$testManifestSigningLaunchSettingsPath";
+        manifestSignerClientDirectory = "$testManifestSignerClientDirectory";
+        manifestSignerClientProjectPath = "$testManifestSignerClientProjectPath";
+        contentTrustRootCaPath = "$testContentTrustRootCaPath";
+        contentTrustRegistryName = "${env:CONTENTTRUST_ADDRESS}";
+        contentTrustSignedImage = "${env:CONTENTTRUST_SIGNEDIMAGE}";
+        contentTrustUnsignedImage = "${env:CONTENTTRUST_UNSIGNEDIMAGE}";
+      }
+  
+      if ('$(nestededge)' -eq 'true')
+      {
+        $context['hostname'] = '$(deviceName)'
+        $context['parentHostname'] = '$(parentName)'
+        $context['parentDeviceId'] = '$(parentDeviceId)'
+  
+        if ('$(test_type)' -eq 'nestededge_isa95')
+        {
+          $context['deviceId'] = '$(Lvl3DeviceId)'
+          $context['edgeProxy'] = '$(proxyAddress)'
+          $context['isa95Tag'] = 'true'
+        }
+      }
+  
+      if ('$(arch)' -eq 'arm32v7' -Or '$(arch)' -eq 'arm64v8')
+      {
+        $context['optimizeForPerformance'] = 'false'
+        $context['setupTimeoutMinutes'] = 10
+        $context['teardownTimeoutMinutes'] = 10
+        $context['testTimeoutMinutes'] = 10
+      }
+  
+      if ($env:AGENT_PROXYURL)
+      {
+        $context['testRunnerProxy'] = $env:AGENT_PROXYURL
+        $context['edgeProxy'] = $env:AGENT_PROXYURL
+      }
+      
+      $context | ConvertTo-Json | Out-File -Encoding Utf8 '$(binDir)/context.json'
+      Get-Content -Path '$(binDir)/context.json'
```

Comment:
This is added for testing purpose to see the contents of context.json. Can remove it

Reply:
True. But I have found this to be really useful for debugging.

This isn't our usual notion of "production code" -- it's an internal azure devops pipeline that just prints something in one of many stages, while running E2E tests.

The customer doesn't see this.